end-to-end-test/remote/specs/core/groupComparisonLollipop.spec.js

```diff
@@ -0,0 +1,234 @@
+var assert = require('assert');
+var goToUrlAndSetLocalStorage = require('../../../shared/specUtils')
+    .goToUrlAndSetLocalStorage;
+var waitForNetworkQuiet = require('../../../shared/specUtils')
+    .waitForNetworkQuiet;
+var setInputText = require('../../../shared/specUtils').setInputText;
+var setSettingsMenuOpen = require('../../../shared/specUtils')
+    .setSettingsMenuOpen;
+const { getElementByTestHandle } = require('../../../shared/specUtils');
+
+const CBIOPORTAL_URL = process.env.CBIOPORTAL_URL.replace(/\/$/, '');
+
+describe('group comparison mutations tab tests', function() {
+    describe('lollipop alerts and plot display', function() {
+        before(function() {
+            goToUrlAndSetLocalStorage(
+                `${CBIOPORTAL_URL}/comparison?comparisonId=634006c24dd45f2bc4c3d4aa`
+            );
+            $('div[data-test="ComparisonPageOverlapTabDiv"]').waitForExist();
+        });
+
+        it('too many groups alert displayed when more than 2 groups selected', function() {
+            $('a.tabAnchor_mutations').waitForExist();
+            $('a.tabAnchor_mutations').click();
+            browser.pause(1000);
+            assert.equal(
+                $('[data-test="TooManyGroupsAlert"]').isDisplayed(),
+                true
+            );
+        });
+
+        it('not enough groups alert displayed when less than 2 groups selected', function() {
+            $('a=Deselect all').waitForExist();
```

Comment:
you can probably assume that this button exists based on the wait in the before callback